src/tree.ts

```diff
@@ -206,19 +184,24 @@ const makeSrcSet = (
     return createFileLink(handle, transformOptions, options.useValidator)();
   }
 
-  const resolutions: any[] = R.map(R.ifElse(
-    R.is(Number),
-    (n: number) => `${n}w`,
-    R.identity,
-  ), options.resolutions);
+  const resolutions = options.resolutions.map((val: any) => {
+    const resString = typeof val === 'number' ? `${val}w` : val;
+    return resString;
+  });
 
-  const widths = R.map(getWidth(width), options.resolutions);
+  const widths = options.resolutions.map((val: any) => {
+    return getWidth(width)(val);
+  });
 
-  const urls: any[] = mapIndexed((width: number, index: number) => {
+  const urls: any[] = widths.map((width: number, index: number) => {
     return createFileLink(handle, transformOptions, options.useValidator, index)(width);
   }, widths);
 
-  return R.join(', ', R.map(R.join(' '), R.zip(urls, resolutions)));
+  const urlWithReso = urls.map((url, index) => {
```

Comment:
use one liner instead and return joinded array instead of assigining it to variable